### DISCUSSION TITLE
Request for Imagenet annotation file train_1percent.txt and train_10precent.txt

### DISCUSSION THREAD

**Discussion by mZhenz** [open-mmlab/mmselfsup#531] [General] [answered]
**Checklist**

1. I have searched related issues but cannot get the expected help.
2. I have read the FAQ documentation but cannot get the expected help.

Hi, I want to train Imagenet with only 1% and 10% data with the configs files in configs/benchmark/imagenet/imagenet_1percent/. Could you offer the annotation file train_1percent.txt and train_10precent.txt? Or how can I get these files? Thanks a lot.

**Comment by fangyixiao18** [comment, marked as answer]
You can check this file: https://github.com/open-mmlab/mmselfsup/blob/master/tools/data_converters/convert_imagenet_subsets.py

At the begining of the file, we provide the google's link of the subset.

**Comment by mZhenz** [comment]
thanks a lot.
